assets/js/phoenix_live_view/dom.js

```diff
@@ -60,7 +60,9 @@ let DOM = {
 
   wantsNewTab(e){
```

Comment:
I wonder if this function should have a more accurate name? Or perhaps 3 separate explicit functions?